src/components/victory-bar.jsx

```diff
@@ -184,6 +190,7 @@ class VBar extends React.Component {
       x: this.createStringMap(props, "x"),
       y: this.createStringMap(props, "y")
     };
+    this.data = this.getData(props);
```

Comment:
Is `this.data` used anywhere? I would expect it to be used inside `consolidateData` to provide some default data.